spec/signin/Checklist_Sidebar.txt

```diff
@@ -0,0 +1,21 @@
+Sidebar
+1. Verify that 'Tops' filter tab is opens
+2. Verify that 'Dresses' filter tab is open
+3. Verify that 'Tops' filter tab select
+4. Verify that 'Dresses' tab filter select
+5. Verify that 'Size' filter tab is select
+6. Verify that 'Color' filter tab is select
+7. Verify that 'Compositions' filter tab is select
+8. Verify that 'Styles' filter tab is select
+9. Verify that 'Properties' filter tab is select
+10. Verify that 'Availability' filter tab is select
+11. Verify that 'Manufacturer' filter tab is select
+12. Verify that 'Condition' filter tab is select
+13. Verify that user can edit 'Price' by range filter
```

Comment:
There are also two options - low price and max price.
Are there default values applied when user didn't set any price range?
Verify on setting lower range of price to $16, only items with price over $16 are displayed
Also You can check for the minimum and max range values, what are they dependant on?